packages/engine/crawlers/sui/collectibles.ts

```diff
@@ -0,0 +1,151 @@
+import type {
+	JsonRpcProvider,
+	PaginatedObjectsResponse,
+	SuiObjectData,
+} from '@mysten/sui.js';
+import { Networks } from '@walless/core';
+import type { CollectibleDocument } from '@walless/store';
+
+export const getSuiCollectibles = async (
+	connection: JsonRpcProvider,
+	owner: string,
+) => {
+	const collectibleDocuments: CollectibleDocument[] = [];
+	let objectResponses: PaginatedObjectsResponse['data'] = [];
+	let nextCursor: PaginatedObjectsResponse['nextCursor'] | undefined;
+	let cursor: PaginatedObjectsResponse['nextCursor'] | undefined;
```

Comment:
Do we need to mark it undefine when declare uninitialized variables

Reply:
`cursor` is needed to be undefined for being used in the first `while` loop, avoid the `used before being assigned` error